.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,17 +2,19 @@
 🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨
 I ACKNOWLEDGE THE FOLLOWING BEFORE PROCEEDING:
 My Pull Request follows the contributing guidlines, available at
-https://github.com/fluenthub-community/FluentHub/blob/main/.github/CONTRIBUTING.md
+https://github.com/fluenthub-community/FluentHub/blob/main/.github/CONTRIBUTING.md;
+My Pull Request follows the FluentHub Pull Request Guidelines (as marked in the commented fields);
+The above 'I ACKNOWLEDGE THE FOLLOWING BEFORE PROCEEDING' is instantaneously bound to the author of this Pull Request upon creation.
 -->
 
 ### ⏱️ Before you start
-<!-- Put an 'x' in the brackets to tick -->
+<!-- Put an 'x' in the brackets to tick or a space to leave clear -->
 
-- [ ] Have you checked if a similar PR has already been requested? 
-- [ ] Have you built and ran the app?
+- [] Have you checked if a similar PR has already been requested? 
+- [] Have you built and ran the app?
```

Comment:
Why did you come up with the idea of removing these spaces?

Reply:
I think it is because it renders normally:
- []
But if someone marks it complete with an x, they don't need to remove a space.

Reply:
> Why did you come up with the idea of removing these spaces?

@arihant2math is right. I did that also because statistically people would not know how to use that feature of markdown correctly and it would format incorrectly.

Reply:
@DeveloperWOW64 Undersood and it's already solved.